src/main/java/by/lawaksoft/tradebot/exception/entity/BusinessException.java

```diff
@@ -0,0 +1,36 @@
+package by.lawaksoft.tradebot.exception.entity;
```

Comment:
could you move to dto

it isn't entity exception, how I understood

Reply:
move to exception.dto package